lib/OpenQA/Worker/Common.pm

```diff
@@ -29,13 +29,13 @@ our @EXPORT = qw($job $verbose $instance $worker_settings $pooldir $nocleanup
   $hosts $ws_to_host $current_host
   $worker_caps $testresults update_setup_status
   STATUS_UPDATES_SLOW STATUS_UPDATES_FAST
-  add_timer remove_timer change_timer get_timer
+  add_timer remove_timer change_timer get_timer get_timers
   api_call verify_workerid register_worker ws_call);
 
 # Exported variables
 our $job;
 our $verbose  = 0;
-our $instance = 'manual';
+our $instance = 99;
```

Comment:
This means that now it is functional (hopefully) when `--instance` option is not passed to the worker script. It is by no means a limit, it's just different enough (IMO) for systems where you expect to run user started workers.